OpenSSL/test/test_crypto.py

```diff
@@ -13,7 +13,9 @@
 from subprocess import PIPE, Popen
 from datetime import datetime, timedelta
 
-from six import u, b, binary_type
+from six import u, b, binary_type, PY3
+from warnings import simplefilter
```

Comment:
warnings is a stdlib module therefore the import should be in the previous block